chain/store/statedb.go

```diff
@@ -92,6 +92,144 @@ func (sdb *StateDB) IntermediateRoot() (common.Uint256, error) {
 	return sdb.trie.Hash(), nil
 }
 
+func (sdb *StateDB) PruneStatesLowMemory(full bool) error {
+	if full {
+		log.Infof("Start pruning...")
```

Comment:
If so this will be printed for every block, very annoying during block syncing. But we can probably add another argument like `verbose` to control log behavior